### ISSUE TITLE
Many Gregtech materials with standard smelting recipes cannot be smelted in the Infernal Kiln

### ISSUE THREAD

**Issue by Meta-Didact** [FTBTeam/FTB-Modpack-Issues#6000]
### Modpack

FTB Interactions Remastered

### Modpack version

1.6.0

### Launcher

Prism

### Has the pack been modified

No

### Describe the bug

Many (Most) Gregtech dusts and crushed/centrifuged ores cannot be smelted in the Infernal Kiln, even if they can be smelted in something that implements vanilla smelting.

Most raw ores can be smelted but not the products of _most_ GT ore processing. 

For example, the kiln can smelt copper ingots from ores _and_ purified crushed copper ore **but nothing else**. Tin ingots can be produced from ores or crushed _cassiterite_ ore (not purified), and nothing else. Iron can be produced from ore, every other mod's pulverized iron, or various stages of brown limonite or basaltic mineral sand processing. Gold seems to be almost completely covered but it is an exception.

### Steps to reproduce

Attempt to smelt most Gregtech materials that have ordinary smelting recipes in the Infernal Kiln

### Expected behaviour

Infernal Kiln multiblock should be able to process all basic smelting recipes, per its quest description.

### Additional information

#5772 solved some issues but overall left the recipes in a wildly inconsistent state as to what can be smelted or not. That issue was closed before I commented on the incompleteness of the fix.

**Comment by ArtpokeBr** [comment]
![image](https://github.com/user-attachments/assets/e6e6cfcb-d9db-47a7-be97-2eec471cd3d2)
Yeah, its kinda Weird, as **it should** get all furnace recipes and port them to the kiln

**Comment by ArtpokeBr** [comment]
Yeah just found out that this happens beacuse GT Register's its furnace recipes after Vanilla Furnace Recipes so the infernal kiln code cannot import it.

I tried some things but I couldn't manage to make it work, what makes it hard is how GTCEU has moved away from CraftTweaker but Multiblocked still uses CT for coding recipes, if Multiblocked used GroovyScript I think this would've been much easier to fix...

I'll leave this issue open as maybe one day I would be able to find a solution for this.
